nava/__main__.py

```diff
@@ -13,10 +12,38 @@ def main():
 
     :return: None
     """
-    args = sys.argv
-    tprint("nava")
-    tprint("V:" + NAVA_VERSION)
-    nava_help()
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
```

Comment:
@AHReccese This argument contains an issue!

After testing the code in **Windows 11** with **Python 3.9**, it was found that the function returned a **list** instead of a **string**, resulting in an error.

```bash
nava --file=test.wav
```

There are two ways in which we can handle it:

1. Remove `nargs=?` for this argument
2. Using `file_name = args.file[0]` instead of `file_name = args.file` (line 39)

Please check it out 🔥 
Thanks

Reply:
I think the first option is the better one. @AHReccese

Reply:
Hey Guys, My HUGE apologies for the latency, I hope it never happens again.
you know, shit happens and I am in a real shit these days. 

It is because the `nargs` parameter for the `--file` input was set to 1 instead of `?`.
`?` means either zero or one which is the case in our program and it is either None or a single value(not a list).
It is solved now.
I also added a `-v` flag with the same functionality as `--version`, it is very common as you can see in many CLI packages like `Yarn`, `NPM`, and ...

@sepandhaghighi @sadrasabouri